pom.xml

```diff
@@ -12,13 +12,12 @@
 
 	<properties>
 		<!-- OUR VERSION -->
-		<revision>1.0-SNAPSHOT</revision>
+		<revision>1.0.0-SNAPSHOT</revision>
 
 		<!-- DEPENDENCIES VERSION -->
 		<jdk.version>1.8</jdk.version>
-		<cds.services.version>1.7.0</cds.services.version>
-		<spring.boot.version>2.3.1.RELEASE</spring.boot.version>
-
+		<cds.services.version>1.8.1</cds.services.version>
+		<spring.boot.version>2.2.6.RELEASE</spring.boot.version>
```

Comment:
Recommended Spring version for 1.8.1 was `2.3.2.RELEASE`, this change seems to lower the Spring version. Is this intended?

Reply:
It might have been an error during my merge. I am updating to 2.3.2.RELEASE version.